core/application/app_test.go

```diff
@@ -454,7 +461,7 @@ func TestApp_GetAllMicroVM(t *testing.T) {
 			expect: func(rm *mock.MockMicroVMRepositoryMockRecorder, em *mock.MockEventServiceMockRecorder, im *mock.MockIDServiceMockRecorder, pm *mock.MockMicroVMServiceMockRecorder) {
 				rm.GetAll(
 					gomock.AssignableToTypeOf(context.Background()),
-					gomock.Eq("default"),
+					gomock.Eq(models.ListMicroVMQuery{"namespace": "default"}),
```

Comment:
could not click on line 457 above, but do we want the name version of "microvms exist in namespace and are returned"